1_slices_modules_and_objects/live_coding/zoo_2/src/main.rs

```diff
@@ -19,7 +19,7 @@ fn main() {
         arthur.scream();
     }
 
-    bob.take_care(&[&mut arthur, &mut an_ape]);
+    bob.take_care(&mut [&mut arthur, &mut an_ape]);
```

Comment:
I'm really happy that Bob decided to properly take care of me :) I'll review your whole exercise tomorrow!